tahrir_api/dbapi.py

```diff
@@ -976,6 +976,39 @@ def add_authorization(self, badge_id, person_email):
 
         return False
 
+    @autocommit
+    def delete_authorization(self, badge_id, person_email):
+        """
+        Delete an authorization (remove someone's admin rights for a certain badge)
+
+        :type badge_id: str
+        :param badge_id: ID of the badge
+
+        :type person_email: str
+        :param person_email: Email of the Person to remove rights from
+        """
+
+        if not self.person_exists(email=person_email):
+            return False
+
+        if not self.badge_exists(badge_id):
+            return False
+
+        person = self.get_person(person_email)
```

Comment:
Are those two queries actually necessary? If the person does not exist, `get_person()` will return `None`, and if the `badge_id` does not exist then the query for the Authorization will not find anything and the function will return `False` anyway. You could do something like:

```suggestion
        person = self.get_person(person_email)
        if not person:
            return False
```